website/docs/developer/deployment-guide/creating-orca-archive-bucket.md

```diff
@@ -275,3 +275,14 @@ Replace the number `000000000000` with your DR account number.
 The Resource value is the bucket and bucket paths that the Cumulus application
 can access. Replace `PREFIX-orca-reports` with the name
 of the Orca reports bucket created in the previous section.
+
```

Comment:
Do we need to update the migration notes as well even though I think everyone has already migrated to a latest version.
https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md#migration-notes-3

Reply:
My plan is to deploy the updated documentation and post a message in Slack to our customers informing them of the fix they'll need to make.